Services/Common/OJS.Workers/OJS.Workers.Compilers/Compiler.cs

```diff
@@ -61,6 +68,7 @@ public abstract class Compiler : ICompiler
 
         this.CompilationDirectory = Path.Combine(inputFileDirectory, CompilationDirectoryName);
         Directory.CreateDirectory(this.CompilationDirectory);
+        this.logger.LogInformation($"Compilation directory created: {this.CompilationDirectory}");
```

Comment:
How we will know for which submission is created?

Reply:
This log is among others which specify the Id of the current submission. For example: ```Executing submission with id: 293```, etc...

Reply:
All the workers write to the same file. If different submissions are executed in parallel logs might get mixed.